src/dnx.host/RuntimeBootstrapper.cs

```diff
@@ -58,90 +58,97 @@ private static void PrintErrors(Exception ex)
 
         public static Task<int> ExecuteAsync(string[] args, FrameworkName targetFramework)
         {
-            var app = new CommandLineApplication(throwOnUnexpectedArg: false);
```

Comment:
Partly because it was how I was able to actually figure out what was wrong, since the Exception was normally being lost in the ether. I think a useful thing to keep in though so I left it. Without this, if an exception is thrown while booting DNX/Helios, it is lost to the ether. Now, at least, if there is a Console attached it can go there. It's not perfect, but it's something to help debugging when you're running `iisexpress` from the console.